src/commands/createFunctionApp.ts

```diff
@@ -4,14 +4,15 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { AzureTreeDataProvider, IActionContext, IAzureNode, IAzureParentNode } from 'vscode-azureextensionui';
```

Comment:
Minor grievance with your title...  These are by definition not unit tests because you're enabling running against Azure.